src/excalidraw-app/data/index.ts

```diff
@@ -30,7 +30,25 @@ const generateRoomId = async () => {
   return bytesToHexString(buffer);
 };
 
-export const SOCKET_SERVER = process.env.REACT_APP_SOCKET_SERVER_URL;
```

Comment:
This is going to break everyone that currently uses excalidraw from source with their own specific server. Could we keep supporting both? If REACT_APP_SOCKET_SERVER_URL is set then return it otherwise use REACT_APP_PORTAL_URL instead?